### PR TITLE
code review #2

### PR DESCRIPTION
**Pull request by David405**

### CODE REVIEW

**Comment by David405** [comment]
I believe this is ready for one final code review.

File: assets/main.css [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+  margin-top: 3%;
+}
+
+.before_footer_top > div > .h6 {
+  color: #000;
+  font-size: 0.9rem;
+  font-weight: bold;
+  padding: 2% 0 0 5%;
+}
+
```
Comment (stratospherique) [inline assets/main.css]: - You are overwriting default bootstrap classes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1570 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (stratospherique) [inline index.html]: - Use responsive meta tag
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

**Comment by stratospherique** [comment]
## And that's it. When finished implementing the above, kindly resubmit another code review request :wink: . Keep UP the good work :muscle: :muscle: .

File: README.md [hunk 3]
```diff
@@ -1,2 +1,9 @@
...
-Microverse Main curriculum
+# Using-bootstrap
+
+This Project is part of the Microverse student's curriculum. It involves building a clone of the newsweek.com website.
+The website is built using HTML, CSS and the Bootstrap framework.
+
+# Author
+David Asamonye
+
+# Live Preview
```
[the remark this answers is not in the document]
Comment (David405) [reply README.md]: done

File: index.html (hunk 2 above)
Comment (David405) [reply index.html]: done

**Pull request merged**